Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeAMSIConfigurationState.ps1

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+. $PSScriptRoot\..\..\Helpers\Invoke-CatchActions.ps1
+Function Get-ExchangeAMSIConfigurationState {
+
+    begin {
+        Write-Verbose "Calling: $($MyInvocation.MyCommand)"
```

Comment:
In here we should have the default values set for, and remove the additional locations in the catch or where we set them multiple times to avoid additional lines of code

```
$amsiState
$amsiOrgWideSetting
$amsiConfigurationQuerySuccessful
```

Reply:
Done. Will be addressed with the next commit.